sources/parser/utils.c

```diff
@@ -68,20 +68,31 @@ t_token	*process_word_tokens(t_command *cmd, t_token *token, char **env,
 		int last_exit)
 {
 	int					i;
+	int					j;
 	int					count;
 	t_content_params	content_params;
+	t_token				*orig_token;
 
 	count = count_word_tokens(token);
 	cmd->args = malloc(sizeof(char *) * (count + 1));
 	if (!cmd->args)
 		return (token);
+	orig_token = token;
 	i = 0;
 	while (i < count && token && token->type == WORD)
 	{
 		content_params.envs = env;
 		content_params.last_exit_code = &last_exit;
 		content_params.content = token->content;
 		cmd->args[i] = get_token_content_value(&content_params);
+		if (!cmd->args[i])
+		{
+			for (j = 0; j < i; j++)
```

Comment:
Deixaremos o *for* por hora, depois refatoramos?

Mais, pra não esquecer mós devido a norma.

Reply:
Alterei essa parte e quebrei em funcao a parte https://github.com/peda-cos/minishell/pull/46/commits/93fc976443e232c76eb7f72b13c92c6738cd5c33, assim quando formos dar uma geral de revisao de norminette nos preocupamos apenas em separar as funcoes em arquivos.